ibm_vpd_utils.cpp

```diff
@@ -1221,5 +1221,47 @@ void getBackupRecordKeyword(std::string& record, std::string& keyword)
         }
     }
 }
+
+int isReadOnlyFRU(const std::string& vpdPath, const nlohmann::json& jsObject)
+{
+    if (vpdPath.empty())
```

Comment:
This should be common in the main part of the tool not here

Reply:
done